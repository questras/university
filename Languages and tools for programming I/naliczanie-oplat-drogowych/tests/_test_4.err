Error in line 5:  	WAD8405 	 A12 	 120,0 	 100,0 	
Error in line 8:  	W	 A	 	0	 
