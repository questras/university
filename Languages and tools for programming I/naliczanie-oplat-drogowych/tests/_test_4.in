 R1234567890  A12  100,0 
  R1234567890   A12   120,0  
	CARYCA1		S12		10,0	
	 CARYCA1	 	S12	 	20,0	 
 	WAD8405 	 A12 	 120,0 	 100,0 	
		WAD8405 S12 20,0
	 	WAD8405 S12 10,0
 	W	 A	 	0	 
 ? 
	?	WAD8405	
