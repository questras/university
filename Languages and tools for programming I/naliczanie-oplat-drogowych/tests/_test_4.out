CARYCA1 S 10,0
R1234567890 A 20,0
WAD8405 S 10,0
A12 20,0
S12 20,0
WAD8405 S 10,0
